app/serializers/job_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class JobSerializer < ActiveModel::Serializer
   attributes :id, :title, :description, :url, :location, :posted_date, :remote
-  has_one :company
+  has_one :company, serializer: CompanyForJobsSerializer
```

Comment:
Hi Jared, I also don't quite understand why you need a specific JobForCompanySerializer in addition to the JobSerializer. Does it interfere with the reference to the CompanyForJobsSerializer you need on line 3 in JobSerializer?